app/processing/settlement/get-last-settlement.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../../data')
+
+const getLastSettlement = async (settlement, transaction) => {
+  return db.settlement.findOne({
+    transaction,
+    attributes: [
+      'paymentRequestId',
+      'invoiceNumber',
+      'reference',
+      'settled',
+      'settlementDate',
+      'value'
```

Comment:
Question whether all of these are needing to be returned

Reply:
Not as many as I originally thought!  Updated.